packages/core/src/components/CopyTextButton/CopyTextButton.test.tsx

```diff
@@ -55,29 +56,30 @@ const apiRegistry = ApiRegistry.from([
 ]);
 
 describe('<CopyTextButton />', () => {
-  it('renders without exploding', () => {
-    const { getByDisplayValue } = render(
-      wrapInTestApp(
-        <ApiProvider apis={apiRegistry}>
-          <CopyTextButton {...props} />
-        </ApiProvider>,
-      ),
+  it('renders without exploding', async () => {
+    const { getByDisplayValue } = await renderInTestApp(
+      <ApiProvider apis={apiRegistry}>
+        <CopyTextButton {...props} />
+      </ApiProvider>,
     );
     getByDisplayValue('mockText');
   });
 
   it('displays tooltip on click', async () => {
+    jest.useFakeTimers();
     document.execCommand = jest.fn();
-    const rendered = render(
-      wrapInTestApp(
-        <ApiProvider apis={apiRegistry}>
-          <CopyTextButton {...props} />
-        </ApiProvider>,
-      ),
+    const rendered = await renderInTestApp(
+      <ApiProvider apis={apiRegistry}>
+        <CopyTextButton {...props} />
+      </ApiProvider>,
     );
     const button = rendered.getByTitle('mockTooltip');
     fireEvent.click(button);
+    act(() => {
+      jest.runAllTimers();
+    });
     expect(document.execCommand).toHaveBeenCalled();
```

Comment:
you have a lot of `runAllTimers` across the codebase - would it make sense and have a little bit less boilerplate if you did like `await waitFor(() => expect(document.execCommand).toHaveBeenCalled())`?

Reply:
Meh, it's such a nitpick and the contribution is great overall so I'll just merge and leave this for some future optional tweak :)

Reply:
Thanks for pointing that out, I will fix that very soon. 👍🏼